core-runner/src/runner.rs

```diff
@@ -84,12 +84,14 @@ impl Config {
 /// Result of one or more message handling.
 #[derive(Debug, Default, Clone)]
 pub struct RunNextResult {
+    /// How many messages were handled.
+    pub handled: u32,
```

Comment:
```suggestion
```
This should be removed, because we always handle only one message while running.

Reply:
it can be 0

Reply:
Yes, but we need to move message processing to the pallet. This will be done after https://github.com/gear-tech/gear/pull/459